src/userbase-js/types/index.d.ts

```diff
@@ -120,29 +122,29 @@ export interface Userbase {
 
   forgotPassword(params: { username: string }): Promise<void>
 
-  openDatabase(params: { databaseName?: string, databaseId?: string, changeHandler: DatabaseChangeHandler }): Promise<void>
+  openDatabase(params: databaseNameXorId & { changeHandler: DatabaseChangeHandler }): Promise<void>
 
   getDatabases(params?: { databaseName?: string, databaseId?: string }): Promise<DatabasesResult>
```

Comment:
Could be:

`params? : databaseNameXorId`

Reply:
Fixed in commit https://github.com/smallbets/userbase/pull/236/commits/c221285795975a877eaddf6868caa058947ee191